build_tools/embed_data/c_embed_data_test.cc

```diff
@@ -10,21 +10,31 @@
 
 namespace {
 
+#if defined(_MSC_VER)
+#define NEWLINE "\r\n"
+#else
+#define NEWLINE "\n"
+#endif
+
 TEST(Generator, TestContents) {
   auto* toc1 = testembed1_create();
   ASSERT_EQ("file1.bin", std::string(toc1->name));
-  EXPECT_EQ(R"(Are you '"Still"' here?)"
-            "\n",
-            std::string(toc1->data));
+  EXPECT_EQ(R"(Are you '"Still"' here?)" NEWLINE, std::string(toc1->data));
+#if defined(_MSC_VER)
+  EXPECT_EQ(25, toc1->size);
+#else
```

Comment:
I am not sure if it is the best approach to support the test in Windows. I am open to suggestions.

Reply:
Seems fine to me. It's a bit weird that Windows is using `\r\n` instead of `\n`... it looks like we're explicit in writing out just `\n`.